### ISSUE TITLE
ros-overlay in layman overlay registry availability

### ISSUE THREAD

**Issue by archenroot** [ros/ros-overlay#144]
Hi @allenh1 ,

summer time and being busy with few neural network prototypes with help of ROS :-) I think we could add the ros-overlay into layman overlay registry, so all artifacts are also searchable via zugaina.

Could you please provide me with your bugs.gentoo.org email registered, I will fill it into repositories.xml as owner. Gentoo also send you confirmation, etc....

Hope you enjoy summer time :+1:

**Comment by allenh1** [comment]
> summer time and being busy with few neural network prototypes with help of ROS :-) I think we could add the ros-overlay into layman overlay registry, so all artifacts are also searchable via zugaina.

Awesome! Sounds like some cool work!

> Could you please provide me with your bugs.gentoo.org email registered, I will fill it into repositories.xml as owner. Gentoo also send you confirmation, etc....

I think the overlay is already registered, but, if not, this would be great! My registered email is  hunter@openrobotics.org (my OSRF email).

**Comment by archenroot** [comment]
@allenh1 - sorry, I overlooked it, you are right, it is in there...

**Comment by allenh1** [comment]
Not a problem! Thanks for looking into it.

**Comment by allenh1** [comment]
@archenroot If you'd like to update the owner on [this list](https://overlays.gentoo.org/) that would be great!
